lib/inventory.js

```diff
@@ -85,6 +85,20 @@ Inventory.prototype.toBufferWriter = function(bw) {
 };
 
 /**
+ * @return {String} name of inventory type
+ */
+Inventory.prototype.typeName = function () {
+  return Inventory.TYPE_NAME[this.type];
+};
+
+/**
+ * @return {String} name of inventory type : hash
+ */
+Inventory.prototype.toString = function () {
```

Comment:
Generally, all of the `toString()` methods have equivalent `fromString` methods, and can be passed in as an argument to the constructor. This may be better as an `inspect()` method (e.g. https://github.com/bitpay/bitcore-p2p/blob/master/lib/pool.js#L337).